string client_name
string reference_name
string command
string primary_command_spec
string secondary_command_spec
string[] args
---
bool success
int32 exit_code
string error_description
bool is_consistent
string[] queried_objects